src/ipc-dbus/wpantund.conf

```diff
@@ -28,6 +28,10 @@
         <allow send_interface="com.nestlabs.WPANTunnelDriver.PermitJoin"/>
         <allow send_interface="com.nestlabs.WPANTunnelDriver.Status"/>
         <allow send_interface="com.nestlabs.WPANTunnelDriver.List"/>
+
+        <!-- Border Agent -->
+        <allow own="ot.ba"/>
```

Comment:
Oh, I see what you are doing. Sorry I misunderstood. `wpantund` talks to the border agent using this API, this API isn't hosted on `wpantund`.

How does the border agent send unsolicited packets back?

I wonder if we should change the approach to be that the border agent checks in with wpantund and then uses D-Bus signals to indicate received packets, and the border agent simply calls a method to send a packet.

Alternatively, you could implement this feature entirely using properties, without developing any new D-Bus protocol. This would work by creating a new property that would act similarly to the associated spinel property. You'd set it to send packets, and you would receive change notifications for inbound packets.

Thoughts?